src/pybind/mgr/dashboard/controllers/nvmeof.py

```diff
@@ -353,9 +353,10 @@ def create(
             self,
             nqn: str,
             rbd_image_name: str,
+            size: Optional[int] = None,
+            rbd_image_size: Optional[int] = None,
```

Comment:
This is mostly for the generated API docs (with the current approach the default doesn't appear anywhere):
```suggestion
            size: Optional[int] = 1024,
            rbd_image_size: Optional[int] = None,
```